sccm/apps/deploy-use/create-deploy-scripts.md

```diff
@@ -57,9 +57,12 @@ By default, users cannot approve a script they have authored. Because scripts ar
 3. In the list of sites, choose your site and then, on the **Home** tab, in the **Sites** group, click **Hierarchy Settings**.
 4. On the **General** tab of the **Hierarchy Settings Properties** dialog box, clear the checkbox **Do not allow script authors to approve their own scripts**.
 
+>[!IMPORTANT]
+>As a Best Practice you should not allow a script author to approve their own scripts! This should only be allowed in a Lab setting. Please carefully consider the potential impact of changing this setting in a Production environment. 
```

Comment:
+>[!IMPORTANT] 
+>As a best practice, you shouldn't allow a script author to approve their own scripts. This should only be allowed in a lab setting. Please carefully consider the potential impact of changing this setting in a production environment.

Reply:
just changing the caps... :)